node/resolvers/catalog/index.ts

```diff
@@ -25,6 +25,25 @@ const extractSlug = item => {
   return item.criteria? `${href[3]}/${href[4]}` : href[3]
 }
 
+/**
+ * Filter the list of products by the available quantity in stock
```

Comment:
> Filter that list products by the available quantity in stock

Reply:
That? Which one? Wouldn't it be better to use: Filter a list...